client/src/main.js

```diff
@@ -21,10 +22,20 @@ if (!window.SANDTANK_SESSION_MANAGER_URL) {
 // ----------------------------------------------------------------------------
 
 Vue.config.productionTip = false;
-
 Vue.use(Vuex);
 Vue.use(Vuetify);
-
+Vue.config.productionTip = false;
```

Comment:
I'm not super familiar with Vue, but wondering if this is needed as the same line is on L24 just above it.

Reply:
Good catch. This is a duplicate. I added it for the tag, but didn't notice is was already there.